neuro_extras/main.py

```diff
@@ -217,6 +222,54 @@ def image_copy(source: str, destination: str) -> None:
     run_async(_copy_image(source, destination))
 
 
+@storage.command("copy")
+@click.argument("source")
+@click.argument("destination")
+def cluster_copy(source: str, destination: str) -> None:
+    run_async(_copy_storage(source, destination))
+
+
+async def _copy_storage(source: str, destination: str) -> None:
+    src_uri = URL(source)
+    src_cluster = src_uri.host
+    src_path = src_uri.parts[2:]
+
+    dst_uri = URL(destination)
+    dst_cluster = dst_uri.host
+    dst_path = dst_uri.parts[2:]
+
+    assert src_cluster
+    assert dst_cluster
```

Comment:
let's have a bit more validation on URIs -- it's traditionally the hard part of the platform.

Reply:
I now use URL normalization/validation code from the client